package.json

```diff
@@ -112,6 +112,7 @@
     "sprintf-js": "1.1.1",
     "standard-loader": "6.0.1",
     "style-loader": "0.20.2",
+    "text-encoder-lite": "git://github.com/coolaj86/TextEncoderLite.git#e1e031b",
```

Comment:
DANG that's annoying... update the to the version we need... but don't release it, 1.0.2 looks like what we want... but its nowhere cept on the commit...

@himdel yah ok with rewording this as follows? `    "text-encoder-lite": "coolaj86/TextEncoderLite#e1e031b",`